inductiva/coastal/test.py

```diff
@@ -0,0 +1,25 @@
+"""End to end CoastalArea scenario simulation"""
```

Comment:
This should not be uploaded as a test to the package, because it will fail due to not having a key.